### ISSUE TITLE
Error starting websocket connection

### ISSUE THREAD

**Issue by CJ1169** [lucent-sea/Remotely#121]
**Describe the bug**
`Error starting websocket connection`

**To Reproduce**

1. Clear `/usr/local/bin/Remotely` for fresh install
2. Download from (self-hosted) server: "Linux x64 Bash Installer"
3. Remove dependencies/ubuntu specific setup as already satisfied. (lines 14 to 33 in script)
4. Run script w/ bash+sudo - host/org already filled in since I downloaded it from host.
5. Script ends; seemingly installed correctly.
6. `sudo systemctl enable/start remotely-agent.service`

**Expected behavior**

Expected behavior would be that the agent connects to server and registers as a device.

**Screenshots**

`journalctl -eu remotely-agent.service`:
```
Aug 31 11:20:39 systemd[1]: Started The Remotely agent used for remote access..
Aug 31 11:20:39 Remotely_Agent[143662]: Service Updater: Version is current.
Aug 31 11:20:40 Remotely_Agent[143662]: Error starting websocket connection.
Aug 31 11:20:40 Remotely_Agent[143662]: Could not find the drive '/-'. The drive might not be ready or might not be mapped.
```
**Desktop (please complete the following information):**
 - OS: Arch Linux (up to date)
 - Browser n/a
 - Version: Remotely 2020.08.20.0804

**Additional context**

Server accepts other devices - that part seems properly set up.

**Comment by lucent-sea** [comment]
Fixed in commit 490f936aca1f7e0aebaa95c1aa35429d049fdbf9.  I can't be sure if something else won't break on Arch Linux, though.  I've never tested on it.

**Comment by CJ1169** [comment]
I can confirm this was fixed on the latest build
